### ISSUE TITLE
Cannot build using APM on `«Windows 10 (^^)»`

### ISSUE THREAD

**Issue by StatelessCat** [forivall/atom-hard-wrap#29]
Cannot build using `APM` on `«Windows 10 (^^)»`.
Seems to be another `node-gyp` issue...
Any idea or workaround ?
Where can I found interesting logs about this issue ? 
Thanks.
R.

**Comment by lalomartins** [comment]
```
> apm i --verbose hard-wrap
Installing hard-wrap to C:\Users\Lalo\.atom\packages REQUEST { url: 'https://atom.io/api/packages/hard-wrap',
  json: true,
  retries: 4,
  strictSSL: true,
  headers: { 'User-Agent': 'npm/3.10.5 node/v4.4.5 win32 ia32' },
  callback: [Function],
  method: 'GET' }
REQUEST make request https://atom.io/api/packages/hard-wrap
REQUEST onRequestResponse https://atom.io/api/packages/hard-wrap 200 { server: 'Cowboy',
  connection: 'close',
  date: 'Mon, 27 Mar 2017 13:56:23 GMT',
  status: '200 OK',
  'x-frame-options': 'DENY',
  'x-xss-protection': '1; mode=block',
  'x-content-type-options': 'nosniff',
  'strict-transport-security': 'max-age=631152000',
  'content-security-policy': 'default-src \'self\'; connect-src \'self\'; font-src https://github-atom-io-herokuapp-com.global.ssl.fastly.net; frame-src \'self\' https://www.youtube.com; img-src https://* \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net data:; media-src \'none\'; object-src \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net; script-src \'self\' \'unsafe-inline\' https://ssl.google-analytics.com https://www.google-analytics.com https://platform.twitter.com https://github-atom-io-herokuapp-com.global.ssl.fastly.net; style-src \'self\' \'unsafe-inline\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net;',
  'content-type': 'application/json; charset=utf-8',
  etag: 'W/"6368b26d980a8252d497ee007ea18eb5"',
  'cache-control': 'max-age=0, private, must-revalidate',
  'x-request-id': '04e8b876-52e3-4bd5-8840-a8c472b72f8a',
  'x-runtime': '0.032400',
  'x-rack-cache': 'miss',
  vary: 'Origin',
  via: '1.1 vegur' }
REQUEST reading response's body
REQUEST finish init function https://atom.io/api/packages/hard-wrap
REQUEST response end https://atom.io/api/packages/hard-wrap 200 { server: 'Cowboy',
  connection: 'close',
  date: 'Mon, 27 Mar 2017 13:56:23 GMT',
  status: '200 OK',
  'x-frame-options': 'DENY',
  'x-xss-protection': '1; mode=block',
  'x-content-type-options': 'nosniff',
  'strict-transport-security': 'max-age=631152000',
  'content-security-policy': 'default-src \'self\'; connect-src \'self\'; font-src https://github-atom-io-herokuapp-com.global.ssl.fastly.net; frame-src \'self\' https://www.youtube.com; img-src https://* \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net data:; media-src \'none\'; object-src \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net; script-src \'self\' \'unsafe-inline\' https://ssl.google-analytics.com https://www.google-analytics.com https://platform.twitter.com https://github-atom-io-herokuapp-com.global.ssl.fastly.net; style-src \'self\' \'unsafe-inline\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net;',
  'content-type': 'application/json; charset=utf-8',
  etag: 'W/"6368b26d980a8252d497ee007ea18eb5"',
  'cache-control': 'max-age=0, private, must-revalidate',
  'x-request-id': '04e8b876-52e3-4bd5-8840-a8c472b72f8a',
  'x-runtime': '0.032400',
  'x-rack-cache': 'miss',
  vary: 'Origin',
  via: '1.1 vegur' }
REQUEST end event https://atom.io/api/packages/hard-wrap
REQUEST has body https://atom.io/api/packages/hard-wrap 3394
REQUEST emitting complete https://atom.io/api/packages/hard-wrap
REQUEST { url: 'https://www.atom.io/api/packages/hard-wrap/versions/1.1.0/tarball',
  strictSSL: true,
  headers: { 'User-Agent': 'npm/3.10.5 node/v4.4.5 win32 ia32' },
  method: 'GET',
  callback: undefined }
REQUEST make request https://www.atom.io/api/packages/hard-wrap/versions/1.1.0/tarball
REQUEST onRequestResponse https://www.atom.io/api/packages/hard-wrap/versions/1.1.0/tarball 302 { server: 'Cowboy',
  connection: 'close',
  date: 'Mon, 27 Mar 2017 13:56:24 GMT',
  status: '302 Found',
  'x-frame-options': 'DENY',
  'x-xss-protection': '1; mode=block',
  'x-content-type-options': 'nosniff',
  'strict-transport-security': 'max-age=631152000',
  'content-security-policy': 'default-src \'self\'; connect-src \'self\'; font-src https://github-atom-io-herokuapp-com.global.ssl.fastly.net; frame-src \'self\' https://www.youtube.com; img-src https://* \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net data:; media-src \'none\'; object-src \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net; script-src \'self\' \'unsafe-inline\' https://ssl.google-analytics.com https://www.google-analytics.com https://platform.twitter.com https://github-atom-io-herokuapp-com.global.ssl.fastly.net; style-src \'self\' \'unsafe-inline\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net;',
  location: 'https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0',
  'content-type': 'text/html; charset=utf-8',
  'cache-control': 'no-cache',
  'x-request-id': '065e4348-558a-4711-8e89-c6978babd00e',
  'x-runtime': '0.078183',
  'x-rack-cache': 'miss',
  vary: 'Origin',
  via: '1.1 vegur' }
REQUEST redirect https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0
REQUEST redirect to https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0
REQUEST {}
REQUEST make request https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0
REQUEST response end https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0 302 { server: 'Cowboy',
  connection: 'close',
  date: 'Mon, 27 Mar 2017 13:56:24 GMT',
  status: '302 Found',
  'x-frame-options': 'DENY',
  'x-xss-protection': '1; mode=block',
  'x-content-type-options': 'nosniff',
  'strict-transport-security': 'max-age=631152000',
  'content-security-policy': 'default-src \'self\'; connect-src \'self\'; font-src https://github-atom-io-herokuapp-com.global.ssl.fastly.net; frame-src \'self\' https://www.youtube.com; img-src https://* \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net data:; media-src \'none\'; object-src \'self\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net; script-src \'self\' \'unsafe-inline\' https://ssl.google-analytics.com https://www.google-analytics.com https://platform.twitter.com https://github-atom-io-herokuapp-com.global.ssl.fastly.net; style-src \'self\' \'unsafe-inline\' https://github-atom-io-herokuapp-com.global.ssl.fastly.net;',
  location: 'https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0',
  'content-type': 'text/html; charset=utf-8',
  'cache-control': 'no-cache',
  'x-request-id': '065e4348-558a-4711-8e89-c6978babd00e',
  'x-runtime': '0.078183',
  'x-rack-cache': 'miss',
  vary: 'Origin',
  via: '1.1 vegur' }
REQUEST onRequestResponse https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0 200 { 'content-length': '8568',
  'access-control-allow-origin': 'https://render.githubusercontent.com',
  'content-security-policy': 'default-src \'none\'; style-src \'unsafe-inline\'',
  'strict-transport-security': 'max-age=31536000',
  vary: 'Authorization,Accept-Encoding',
  'x-content-type-options': 'nosniff',
  'x-frame-options': 'deny',
  'x-xss-protection': '1; mode=block',
  etag: '"f3f5d843d718091d6c806d47950e96a89eb705b2"',
  'content-type': 'application/x-gzip',
  'content-disposition': 'attachment; filename=forivall-atom-hard-wrap-v1.1.0-0-gf3f5d84.tar.gz',
  'x-geo-block-list': '',
  date: 'Mon, 27 Mar 2017 13:56:25 GMT',
  'x-github-request-id': 'C63E:1DF0:4B9F1E:60E649:58D91A08',
  connection: 'close' }
REQUEST finish init function https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0
REQUEST response end https://codeload.github.com/forivall/atom-hard-wrap/legacy.tar.gz/v1.1.0 200 { 'content-length': '8568',
  'access-control-allow-origin': 'https://render.githubusercontent.com',
  'content-security-policy': 'default-src \'none\'; style-src \'unsafe-inline\'',
  'strict-transport-security': 'max-age=31536000',
  vary: 'Authorization,Accept-Encoding',
  'x-content-type-options': 'nosniff',
  'x-frame-options': 'deny',
  'x-xss-protection': '1; mode=block',
  etag: '"f3f5d843d718091d6c806d47950e96a89eb705b2"',
  'content-type': 'application/x-gzip',
  'content-disposition': 'attachment; filename=forivall-atom-hard-wrap-v1.1.0-0-gf3f5d84.tar.gz',
  'x-geo-block-list': '',
  date: 'Mon, 27 Mar 2017 13:56:25 GMT',
  'x-github-request-id': 'C63E:1DF0:4B9F1E:60E649:58D91A08',
  connection: 'close' }
gyp info it worked if it ends with ok
gyp verb cli [ 'C:\\Users\\Lalo\\AppData\\Local\\atom\\app-1.15.0\\resources\\app\\apm\\bin\\node.exe',
gyp verb cli   'C:\\Users\\Lalo\\AppData\\Local\\atom\\app-1.15.0\\resources\\app\\apm\\node_modules\\node-gyp\\bin\\node-gyp.js',
gyp verb cli   'install',
gyp verb cli   '--runtime=electron',
gyp verb cli   '--target=1.3.13',
gyp verb cli   '--dist-url=https://atom.io/download/electron',
gyp verb cli   '--arch=ia32',
gyp verb cli   '--ensure',
gyp verb cli   '--verbose' ]
gyp info using node-gyp@3.4.0
gyp info using node@4.4.5 | win32 | ia32
gyp verb command install []
gyp verb install input version string "1.3.13"
gyp verb install installing version: iojs-1.3.13
gyp verb install --ensure was passed, so won't reinstall if already installed
gyp verb install version is already installed, need to check "installVersion"
gyp verb got "installVersion" 9
gyp verb needs "installVersion" 9
gyp verb install version is good
1.3.13
gyp info ok
[..................] - :
```

then it just sits there, no matter how long I wait.

**Comment by KevinGroarke** [comment]
In case anyone else runs into this issue I solved it by using a previous version of this package. 
`apm install hard-wrap@1.0.1`

**Comment by Ajedi32** [comment]
This is probably a duplicate of #18

**Comment by forivall** [comment]
inactive, and probably solved by merging #18
